.github/workflows/ci.yml

```diff
@@ -5,8 +5,9 @@ on:
     branches: [dev, stg, master]
     types: [opened, reopened, synchronize, edited]
 env:
+  DOCKER_BUILDKIT: 1
```

Comment:
```suggestion
```

I don't think we need this if we're using the new buildx actions.
>Buildx builds using the BuildKit engine and does not require DOCKER_BUILDKIT=1 environment variable to start the builds.

([docs](https://docs.docker.com/buildx/working-with-buildx/#build-with-buildx))

Reply:
Closed in #119.